apps/submit-app.md

```diff
@@ -6,26 +6,121 @@ title: Submit an app
 
 # Your app is ready for takeoff?
 
-Yay! You're done with developing your awesome app and you're excited to make it available to our App store? Congrats! We are just as much excited as you are!
+Yay! You're done with developing your awesome app and you're excited to make it available to our Apps & Themes Store? Congrats! We are just as excited as you are!
```

Comment:
instead of "make it available" I would suggest the following:
- "excited to publish it on our.."
- "excited to distribute it on our.."